CreeDictionary/API/models.py

```diff
@@ -357,12 +360,10 @@ def fetch_lemma_by_user_query(user_query: str, **kwargs) -> "CreeAndEnglish":
         # 2. definition containment of the query word
         cree_results: Set[CreeResult] = set()
 
-        # utilize the spell relax in descriptive_analyzer
-        # TODO: use shared.descriptive_analyzer (HFSTOL) when this bug is fixed:
-        # https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/issues/120
```

Comment:
I released `hfstol-1.2.11` for #120, hopefully the bug is fixed
![Screenshot from 2020-03-10 21-07-45](https://user-images.githubusercontent.com/44753941/76378531-38ff3800-6313-11ea-9671-23b10efa7d5b.png)

Reply:
good news: the ci passes